### PR TITLE
fix(releasenotes): update release notes for 2019.09 release

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/2395

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,20 +4,64 @@ section: root
 hideTOC: true
 ---
 
+# 2019.09 release notes (2019-10-29)
+Packages released:
+- @patternfly/patternfly@v2.40.1
```
Comment (dgutride) [inline RELEASE-NOTES.md]: I believe this should be 2.40.2 to be in sync with the build that this will be released in.

Comment (mcoker) [reply RELEASE-NOTES.md]: Updated, and also updated commit message so it will trigger a release.

**Comment by mcoker** [comment]
@rachael-phillips updated the messages to be past tense

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,20 +4,64 @@ section: root
 hideTOC: true
 ---
 
+# 2019.09 release notes (2019-10-29)
+Packages released:
+- @patternfly/patternfly@v2.40.2
+
+## Components
+- **Charts:**
```
Comment (redallen) [inline RELEASE-NOTES.md]: Usually we alphabetize these, but I'll worry about it later.

**Pull request merged**